cardscan-base/src/main/java/com/getbouncer/cardscan/base/SSDOcrDetect.java

```diff
@@ -182,10 +193,46 @@ public synchronized String predict(Bitmap image, Context context) {
             try {
                 return runModel(image);
             } catch (Error | Exception e) {
-                Log.i("ObjectDetect", "runModel exception, retry object detection", e);
+                Log.i("OCR", "runModel exception, retry object detection", e);
```

Comment:
We should standardize our log tags. It's strange to have a log tag `OCR` talking about object detection. Instead, let's use something like `CardScan` or `Bouncer`